components/script/dom/element.rs

```diff
@@ -363,6 +363,19 @@ impl Element {
         !self.overflow_y_is_visible()
     }
 
+    // https://drafts.csswg.org/cssom-view/#scrolling-box
+    fn has_scrolling_box(&self) -> bool {
+        // TODO: scrolling mechanism, such as scrollbar (We don't have scrollbar yet)
+        //       self.has_scrolling_mechanism()
+        self.overflow_x_is_hidden() ||
+        self.overflow_y_is_hidden()
+    }
+
+    fn has_overflow(&self) -> bool {
```

Comment:
This needs a spec link or something? I cannot find its definition anywhere though...

Reply:
There is a spec for it
https://github.com/servo/servo/pull/19127/files#diff-cd38a1325ddd0891af983d7c49716c1c

Reply:
The document only say  `or the element has no overflow`, and don't give any url to explain it. But I thought the overflow happen when scroll length is longer than client's.

Reply:
And in the new test which I create, there is tests for spec it.